nixos/modules/services/web-apps/pds.nix

```diff
@@ -0,0 +1,232 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  cfg = config.services.pds;
+
+  inherit (lib)
+    getExe
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOption
+    escapeShellArgs
+    concatMapStringsSep
+    types
+    literalExpression
+    ;
+
+  pdsadminWrapper =
+    let
+      cfgSystemd = config.systemd.services.pds.serviceConfig;
+    in
+    pkgs.writeShellScriptBin "pdsadmin" ''
+      DUMMY_PDS_ENV_FILE="$(mktemp)"
+      trap 'rm -f "$DUMMY_PDS_ENV_FILE"' EXIT
```

Comment:
Why do we need a temporary file?

Reply:
We need a temp file because we need to combine env vars from module settings and possibly multiple secret files into one that the admin script can work with

Reply:
Cause the script tries to `source` it and it almost always does not exist (the default is `/pds/pds.env` which looks docker-ish) but even if it does we still want to use our variables instead
https://github.com/bluesky-social/pds/blob/main/pdsadmin/account.sh#L6-L7